server/lib/nl/config_builder/base.py

```diff
@@ -32,6 +32,23 @@
 from server.lib.nl.fulfillment.types import SV2Thing
 import server.lib.nl.fulfillment.utils as futils
 
+_SPECIAL_REPLACEMENTS = {
+    " A ": " a ",
+    " By ": " by ",
+    " Of ": " of ",
+    " For ": " for ",
+    " In ": " in ",
+    " As ": " as ",
+    " Or ": " or ",
+    " On ": " on ",
+    " Is ": " is ",
+    " And ": " and ",
+    " To ": " to ",
+    "Covid": "COVID",
+    "- ": "-",
+    " -": "-",
```

Comment:
I slightly wonder if we want to do the ` - ` trimming only when both sides have numbers?  So try a regex replace for that one?

To avoid remove spaces for names like https://autopush.datacommons.org/browser/dc/5klkb2bl7rwe6

Reply:
Hmm. I don't think this behavior is actually correct :) See https://www.grammarbook.com/punctuation/spacing.asp#:~:text=Use%20no%20spaces%20on%20either,hyphens%2C%20click%20on%20Hyphens.%5D There shouldn't be spaces between hyphens. The only exceptions are when hyphens are "hanging hyphens", e.g. https://community.rmahq.org/rmastyleanddesignguide/writingguide/dashesandhyphens?ssopc=1#:~:text=No%20spaces%20are%20used%20between,after%20it%2C%20but%20not%20before. 

In our case, I think we aren't likely to have hanging hyphens in titles, right?

Reply:
I'm not sure if those rules are exactly relevant to the titles we have?  Those seem to be talking about hyphenated words.

Sources like WB/OECD/etc. use `-` in the titles as a delimiter between concepts.  Dropping the space around them can actually change the meaning.

Consider: "Count of Establishment: Monetary Authorities - Central Bank" => There's a hierarchy here Monetary Authorities => Central Bank.  If we did "Monetary Authorities-Central Bank" that seems to imply the 2nd word is a hyphenated word and is misleading.

Reply:
I am only replacing " - " with "-" if surrounded by digits on both ends.